packages/eslint-config-asini/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "extends": "babel",
+  "rules": {
+    "no-confusing-arrow": 0,
+    "max-len": 0
```

Comment:
This drops `space-infix-ops`, since that has since been [added to `eslint-config-babel`](https://github.com/babel/eslint-config-babel/pull/1).